.golangci.toml

```diff
@@ -8,7 +8,9 @@ exclude-use-default = false
 [linters]
 enable-all = true
 disable = [
+  "bodyclose",
```

Comment:
We probably want this linter but since the fixes involve real code changes in some places, we can sort it out as a separate PR.